protocol/contracts/voting/MasterCandidate.sol

```diff
@@ -0,0 +1,191 @@
+//SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/utils/Strings.sol";
+import "@openzeppelin/contracts/utils/Counters.sol";
+import "@openzeppelin/contracts/utils/cryptography/MerkleProof.sol";
+import "@openzeppelin/contracts/utils/cryptography/draft-EIP712.sol";
+
+/**
+ * @title an NFT for each master candidate, with associated endorsements
+ * 
+ * @notice Captures key aspects of temple governance on chain. Namely
+ * 
+ *   - Anyone in temple discord with at least the Templar Role + verified
+ *     wallet can nominate themselves for candidacy.
+ *   - Any user can endorse candidates
+ *   - EIP712 structured data signing to allow 'gasless voting' via a relay
+ * 
+ * Which masters are currently voted in/active in the multisig etc is handled
+ * at the social layer.
+ */
+contract MasterCandidate is ERC721, Ownable, EIP712 {
```

Comment:
It's an NFT, that means candidates can transfer their NFT to other accounts.

on balance, I think this is good. Social layer will mean if they aren't controlling the account the NFT is transferred to (easy to verify), they should be voted out swiftly.

It does mean there isn't any 'centralised admin' if one wants to change their wallet etc.